addon/mixins/in-viewport.js

```diff
@@ -130,6 +130,7 @@ export default Mixin.create({
       });
     } else {
       return scheduleOnce('afterRender', this, () => {
+        this.get('_rAFAdmin').restart();
```

Comment:
Thinking of two options here:

1.  Auto restart (set isRunning to true) when `add` gets invoked.
2.  Have a separate method but call `restart` right before we call `add`.